sources/packages/web/src/components/aest/InstitutionUserSummary.vue

```diff
@@ -0,0 +1,331 @@
+<template>
+  <v-container>
+    <div class="row">
+      <div class="col align-self-start">
+        <span class="category-header-large color-blue">
+          All Users({{ usersListAndCount.totalUsers }})
+        </span>
+      </div>
+      <div>
+        <InputText
+          v-model="searchBox"
+          placeholder="Search User"
+          @keyup.enter="searchUserTable()"
+        />
+      </div>
+      <div>
+        <v-btn
+          @click="searchUserTable()"
+          color="primary"
+          tile
+          class="float-left ml-1"
+        >
+          <font-awesome-icon icon="search" />
+        </v-btn>
+      </div>
+      <div v-if="clientType === ClientIdType.Institution" class="mx-2">
+        <v-btn class="float-right" @click="openNewUserModal()" color="primary">
```

Comment:
We may have to discuss about getting the background and font color got buttons in place.